packages/musap-rn-kms/src/agent/MusapKeyManagerSystem.ts

```diff
@@ -11,8 +11,7 @@ import {
 } from '@sphereon/musap-react-native'
 import { SscdType } from '@sphereon/musap-react-native/src/types/musap-types';
 import {AbstractKeyManagementSystem} from '@veramo/key-manager';
-import { v4 as uuid } from 'uuid';
-
+import {randomUUID} from 'expo-crypto'
```

Comment:
No, that wasn't the reason for the metnion on get-random-values at all. Please keep the v4 uuid. As mentioend I would have expected a peer-dep for expo-crypto and/or react-native-get-random-values

Reply:
The reason was that you were importing a polyfill for crypto overwriting whatever the app already had as crypto implementation. You can do that in your app, but you really should not do that in a lib or SDK module. The lib just needs to be sure that a crypto peer dep is present

Reply:
As discussed on Slack, externalized keyAlias (as mandatory param) removing the need for any uuid